diff --git a/weechat-color.el b/weechat-color.el
index 581d5e4..242eb2c 100644
--- a/weechat-color.el
+++ b/weechat-color.el
@@ -92,7 +92,7 @@ This will look very bland!"
                                 (seq (or ,std ,astd ,ext ,aext)
                                      ","
                                      (or ,std ,astd ,ext ,aext))))
-                   (seq "b" (in "FDB_-#il"))
+                   (seq "b" (in "-FDB_#il"))
                    ""))
           (seq "" ,attr)
           (seq "" ,attr)